### ISSUE TITLE
Possible renaming

### ISSUE THREAD

**Issue by Owez** [scOwez/tinydb#3]
Will be a pain to pull off but may be required in order to differentiate from python's tinydb unfortunately.

**Comment by JxrdanH** [comment]
- NanoDB (uDB)
- miniDB
- KurzDB
- QuarkDB

**Comment by Owez** [comment]
Thanks for suggestions, I'll search it any are taken @jxrdanhughes :smile:

**Comment by marioidival** [comment]
xibataDB

**Comment by Owez** [comment]
Will use KurzDB as it's not taken and sounds decent, "small" in German seems fitting. Won't rename for a while as have other obligations outside of oss currently.

**Comment by aramodi** [comment]
Kurz means short whereas "klein" or "winzig" means "small"

**Comment by JxrdanH** [comment]
> Kurz means short whereas "klein" or "winzig" means "small"

Yeah, however Kurz has a nice sound to it. Although KleinDB sounds alright

**Comment by Owez** [comment]
KleinDB sounds like a last name whereas kurzdb sounds better. This project is dead however and if reworked will become a fast and embedded nosql db
